rosdep/base.yaml

```diff
@@ -1660,7 +1667,14 @@ libpcl-all-dev:
       packages: [pcl]
   macports: [libpcl]
   opensuse: [pcl]
-  ubuntu: [libpcl-1.7-all-dev]
+  ubuntu:
+    precise: [libpcl-1.7-all-dev]
+    quantal: [libpcl-1.7-all-dev]
+    raring: [libpcl-1.7-all-dev]
+    saucy: [libpcl-1.7-all-dev]
+    trusty: [libpcl-1.7-all-dev]
+    utopic: [libpcl-1.7-all-dev]
+    vivid: [libpcl-dev]
```

Comment:
PCL got accepted into Debian, so it will be in Vivid (it's not in Utopic, as you can see here: http://packages.ubuntu.com/search?suite=all&arch=any&searchon=names&keywords=libpcl-dev). Also, Debian policy is to version the lib package according to the SONAME and have the development package unversioned, as you can only install one at the time.